src/catkin_pkg/package_version.py

```diff
@@ -122,6 +145,17 @@ def update_versions(paths, new_version):
         except RuntimeError as rue:
             raise RuntimeError('Could not bump version number in file %s: %s' % (package_path, str(rue)))
         files[package_path] = new_package_str
+        # Update any setup.py files.
+        setup_py_path = os.path.join(path, 'setup.py')
+        if os.path.exists(setup_py_path):
+            with open(setup_py_path, 'r') as f:
+                setup_py_str = f.read()
+            try:
+                new_setup_py_str = _replace_setup_py_version(setup_py_str, new_version)
+            except RuntimeError as exc:
+                raise RuntimeError('Could not bump version number in file %s: %s' % (setup_py_path, str(exc)))
```

Comment:
Is this going to raise an exception for packages that contain a setup.py without a version (e.g. a lot catkin packages in ROS 1)?

Reply:
Ah, that's a good point. I thought about restricting this check to the situation when your built type is `ament_python`, but that information wasn't readily available and didn't want to cut into it too much. Let me look again at it.

Reply:
That's a good point. I don't see anything which makes running the setup.py modifications conditional on the `ament_python` build type but perhaps it should be?